Tools/dea_tools/dask.py

```diff
@@ -41,44 +46,46 @@ def create_local_dask_cluster(spare_mem="3Gb", display_client=True, return_clien
         sys.path.append("../Scripts")
```

Comment:
"Scripts" no longer exists, can we change this to just include the import and not the `sys` bit?